testing/scenario_app/bin/android_integration_tests.dart

```diff
@@ -170,15 +170,20 @@ void main(List<String> args) async {
     });
 
     await step('Running instrumented tests...', () async {
-      final int exitCode = await pm.runAndForward(<String>[
+      final (int exitCode, StringBuffer out) = await pm.runAndCapture(<String>[
         adb.path,
         'shell',
         'am',
         'instrument',
-        '-w', 'dev.flutter.scenarios.test/dev.flutter.TestRunner',
+        '-w',
+        'dev.flutter.scenarios.test/dev.flutter.TestRunner',
       ]);
       if (exitCode != 0) {
-        panic(<String>['could not install test apk']);
+        panic(<String>['instrumented tests failed to run']);
+      }
+      if (out.toString().contains('FAILURES!!!')) {
```

Comment:
If you found this via android documentation then please include a link. If not then carry on.

Reply:
I did not, mostly stack overflow sleuthing

Reply:
Probably worth adding a comment here since it's pretty unorthodox.

Reply:
Done!